content/security.adoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: simplepage
+title: "Jenkins Security"
+---
+
+
+The Jenkins project takes security seriously and makes every possible effort to
+ensure users can adequately secure their automation infrastructure. Under the
+guidance of the Jenkins Security Officer, link:/2.0/[Jenkins 2.0] (and higher)
```

Comment:
Honestly, given the number of people that are running older versions, I will probably keep this line in for at least 6 months.